recipe/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "sis-extras" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sis_extras-{{ version }}.tar.gz
+  sha256: e7e1892ab5eef36e38880dc9b636597dce83162df4d51f3f7f509a7e792e8a73
+  patches:
+    - patches/0001-loosen-Python-and-streamlit-constraints.patch
+
+build:
+  # Strictly, upstream is py==38 and we don't have
+  # snowflake-snowpark-python for 3.12
+  # no streamlit on s390x
+  skip: True  # [py>311 or s390x]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
```

Comment:
```suggestion
    build:
      - patch     # [not win]
      - m2-patch  # [win]
    host:
```

Reply:
Yep, on its way.
I spent most of the time trying to get my code to flag "patches but no patch" up for me